runtime-infra/pn-ipc.yaml

```diff
@@ -845,6 +845,22 @@ Resources:
         DelaySeconds: 1
         AlarmSNSTopicName: !Ref AlarmSNSTopicName
 
+
+  ### Communication between PN-DELIVERY and PN-DELIVERY-PUSH - FOR NEW NOTIFICATION EVENT ###
+  DeliveryPushValidationInputsQueue:
+    Type: AWS::CloudFormation::Stack
+    UpdateReplacePolicy: Retain
+    DeletionPolicy: Retain
+    Properties:
+      TemplateURL: !Sub "${TemplateBucketBaseUrl}/fragments/sqs-queue.yaml"
+      Parameters:
+        QueueName: !Sub '${ProjectName}-delivery_push_validation_inputs'
+        DelaySeconds: 10
```

Comment:
@simonerondinella è voluto questo valore?

Reply:
In questo caso ho copiato la struttura da DeliveryPushInputsQueue (che va a sostituire per i soli messaggi di validazione) definita nello stesso file. Non so però qual è il motivo dei 10 secondi di delay